src/components/sides/TaskInfo.vue

```diff
@@ -305,7 +305,9 @@
             :key="entity.id"
             v-for="entity in Array.from(selectedEntities.values())"
           >
-            {{ entity.full_name }}
+            <span :class="entity.canceled ? 'canceled' : ''">{{
+              entity.full_name
+            }}</span>
```

Comment:
You can replace ternary with object syntax for class binding.

```suggestion
            <span :class="{ canceled: entity.canceled }">{{ entity.full_name }}</span>
```